packages/block-library/src/navigation/edit/index.js

```diff
@@ -332,9 +330,11 @@ function Navigation( {
 	] = useState();
 	const [ detectedOverlayColor, setDetectedOverlayColor ] = useState();
 
-	const handleUpdateMenu = ( menuId ) => {
+	const handleUpdateMenu = ( menuId, focusNavigationBlock = false ) => {
```

Comment:
Nit: I wonder whether the function signature might be clearer with an options object?

```suggestion
	const handleUpdateMenu = ( menuId, { focusNavigationBlock = false } ) => {
```

That way later in the code where we call this function it will be clearer that the 2nd parameter is for focus.